invenio_pidstore/cli.py

```diff
@@ -153,3 +178,24 @@ def dereference_object(object_type, object_uuid, status):
         click.echo(
             '{0.pid_type} {0.pid_value} {0.pid_provider}'.format(found_pid)
         )
+
+
+@pid.command()
+@click.argument('minter', callback=process_minter)
+@click.argument('ids', nargs=-1, required=True)
+@click.option('-l', '--loader',
+              default='invenio_records.api:Record.get_record',
+              callback=loader_import)
+@with_appcontext
+def mint(minter, ids, loader):
+    """Mint objects."""
+    created_ids = []
+    for id_ in ids:
+        try:
+            created_ids.append(minter(id_, loader(id_)).id)
```

Comment:
One of the idea was that we could use it as a "PID reserve" method for your JSON file.

`cat record-without-pid.json | flask pid mint recid > record.json # enhanced with PID`